packages/@aws-cdk/aws-lambda/test/test.layers.ts

```diff
@@ -71,4 +73,24 @@ export = testCase({
 
     test.done();
   },
+
+  'asset metadata is added to the cloudformation resource'(test: Test) {
```

Comment:
should this mention that this test is specific for Lambda Layers as well

Reply:
It's under `test.layers.ts` :-)

Reply:
ah good call!